scripts/update_from_registry.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python
```

Comment:
let's make sure this is a part of https://github.com/aws-cloudformation/cfn-python-lint/pull/1792 as well

Reply:
thoughts on just putting this in `cfn-lint --update-specs` like https://github.com/aws-cloudformation/cfn-python-lint/pull/1682 if we use the hosted [resource provider schemas](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resource-type-schemas.html) to avoid new credential requirements?

Reply:
yea good question.  Any thoughts on how this would scale and the speed of it?  We could download the zips and read them instead of using the APIs (honestly didn't know that existed).  Plus it would help with my rate limiting issues.